LICENSE.md

```diff
@@ -0,0 +1,2 @@
+Most of this repository is licensed under [MIT](https://opensource.org/licenses/MIT),
```

Comment:
Should we actually include the text of MIT license?

Reply:
That's a good question